package.json

```diff
@@ -1,50 +1,52 @@
 {
-  "name": "freshrss",
-  "type": "module",
-  "description": "A free, self-hostable aggregator",
-  "homepage": "https://freshrss.org/",
-  "readmeFilename": "README.md",
-  "bugs": {
-    "url": "https://github.com/FreshRSS/FreshRSS/issues"
-  },
-  "keywords": [
-    "news",
-    "aggregator",
-    "RSS",
-    "Atom",
-    "WebSub"
-  ],
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/FreshRSS/FreshRSS.git"
-  },
-  "license": "AGPL-3.0",
-  "engines": {
-    "node": ">=18"
-  },
-  "scripts": {
-    "eslint": "eslint .",
-    "eslint_fix": "eslint --fix .",
-    "markdownlint": "markdownlint '**/*.md'",
-    "markdownlint_fix": "markdownlint --fix '**/*.md'",
-    "rtlcss": "rtlcss -d p/themes/ && find p/themes/ -type f -name '*.rtl.rtl.css' -delete",
-    "stylelint": "stylelint '**/*.css' && stylelint --custom-syntax postcss-scss '**/*.scss'",
-    "stylelint_fix": "stylelint --fix '**/*.css' && stylelint --fix --custom-syntax postcss-scss '**/*.scss'",
-    "test": "npm run eslint && npm run stylelint && npm run markdownlint",
-    "fix": "npm run rtlcss && npm run stylelint_fix && npm run eslint_fix && npm run markdownlint_fix"
-  },
-  "devDependencies": {
-    "eslint": "^9.39.0",
-    "@eslint/js": "^9.20.0",
-    "globals": "^16.5.0",
-    "markdownlint-cli": "^0.45.0",
-    "neostandard": "^0.12.2",
-    "rtlcss": "^4.3.0",
-    "sass": "^1.93.3",
-    "stylelint": "^16.25.0",
-    "stylelint-config-recommended-scss": "^16.0.2",
-    "stylelint-order": "^7.0.0",
-    "@stylistic/stylelint-plugin": "^4.0.0"
-  },
-  "rtlcssConfig": {}
+	"name": "freshrss",
+	"type": "module",
+	"description": "A free, self-hostable aggregator",
+	"homepage": "https://freshrss.org/",
+	"readmeFilename": "README.md",
+	"bugs": {
+		"url": "https://github.com/FreshRSS/FreshRSS/issues"
+	},
+	"keywords": [
+		"news",
+		"aggregator",
+		"RSS",
+		"Atom",
+		"WebSub"
+	],
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/FreshRSS/FreshRSS.git"
+	},
+	"license": "AGPL-3.0",
+	"engines": {
+		"node": ">=18"
+	},
+	"scripts": {
+		"eslint": "eslint .",
+		"eslint_fix": "eslint --fix .",
+		"markdownlint": "markdownlint '**/*.md'",
+		"markdownlint_fix": "markdownlint --fix '**/*.md'",
+		"rtlcss": "npm run rtlcss:run && npm run rtlcss:clean && npm run rtlcss:imports",
+		"rtlcss:run": "rtlcss -d p/themes/",
+		"rtlcss:clean": "find p/themes/ -type f -name '*.rtl.rtl.css' -delete",
+		"rtlcss:imports": "find p/themes -type f -name '*.rtl.css' -print0 | xargs -0 sed -i -E '/\\.rtl\\.css/! s|^(@import \"[^\"]+)(\\.css\";)$|\\1.rtl\\2|'",
```

Comment:
This part is modified to rewrite the CSS `@import` URLs to RTL.

Reply:
We now lack a more elegant approach to avoid `find p/themes/ -type f -name '*.rtl.rtl.css' -delete`